app/templates/frameworks/_framework_agreement_legal_text_g-cloud-12.html

```diff
@@ -0,0 +1,61 @@
+<h2 class="govuk-heading-m">1.      The Appointment</h2>
+
+
+<p>
+    This section includes terms about the Supplier appointment.</p>
+
+<h3 class="govuk-heading-s">1.1       Appointment</h3>
```

Comment:
1.1 Appointment is an `<h3>` but 1.2, etc are `<p>`, which would indicate that 1.1 describes them all.